circle.yml

```diff
@@ -131,6 +131,14 @@ defaults:
         testeth -t GeneralStateTests/stEWASMTests -- --testpath tests --vm ~/build/src/libhera.so --singlenet Byzantium --singletest "useGas" --evmc engine=wabt
         testeth -t GeneralStateTests/stEWASMTests -- --testpath tests --vm ~/build/src/libhera.so --singlenet Byzantium --singletest "createFromTransaction" --evmc engine=wabt
 
+  test-wavm: &test-wavm
```

Comment:
This needs to be added into the `steps` part of `linux-clang-shared-release`, otherwise it is not executed.

Reply:
Yes, will add it to steps.